source/Geometry/src/Disk.cc

```diff
@@ -9,8 +9,8 @@ Disk::Disk(int id, double zOffset, double zECCentre, double zECHalfLength, const
  zError(        "zError"     , parsedAndChecked()),
  m_innerRadius( "innerRadius", parsedAndChecked()),
  m_outerRadius( "outerRadius", parsedAndChecked()),
- m_bigDelta(    "bigDelta"   , parsedAndChecked()),
- m_rOverlap(    "rOverlap"   , parsedOnly(), 1.),
+ bigDelta(      "bigDelta"   , parsedAndChecked()),
```

Comment:
I see that smallDelta and rOverlap were renamed to their original name from m_smallDelta and m_rOverlap. I think that it was more convenient to keep those member variables names the same as their string counterparts, but since you decided to rename all of them, why did you drop the m_ for these two?